public/js/lib/themes/light-theme.css

```diff
@@ -252,3 +257,94 @@ div.CodeMirror span.eval-text {
   line-height: 1.4em;
   min-height: 1.4em;
 }
+
```

Comment:
We don't use pngs/svgs from m-c. We could patch all the URLs and copy the images over but I don't think it's worth it, as we are rewriting the basic components and using inline SVGs for everything.

But you're right in that I should probably just comment out the URLs instead of removing them, as that's better documentation for how we patched the file.